R/derive_param_clinbenefit.R

```diff
@@ -278,7 +296,7 @@ derive_param_clinbenefit <- function(dataset,
 
   ovr_data <- ovr_data %>%
     filter(
-      !(AVALC %in% c("NA", "NE", "ND", "PD")) & !is.na(AVALC) &
+      AVALC %in% clinben_vals & !is.na(AVALC) &
```

Comment:
very minor, and likely my misunderstanding, but why is `& !is.na(AVALC)` needed here?

Reply:
I think you're right it isn't needed now that the code was changed. `AVALC %in% clinben_vals` takes care of all the necessary checking and would already exclude blanks.

Reply:
`!is.na(ADT)` is not needed because it is implied by `ADT >= !!reference_date + days(ref_start_window)`